test-sbt/jvm/src/test/scala/zio/test/sbt/ZTestFrameworkSpec.scala

```diff
@@ -34,12 +36,13 @@ object ZTestFrameworkSpec {
     test("should correctly display colorized output for multi-line strings")(testColored()),
     test("should test only selected test")(testTestSelection()),
     test("should return summary when done")(testSummary()),
+    test("should use a shared layer without re-initializing it")(testSharedLayers()),
```

Comment:
Are we still using this or are we now just testing by having those two tests in the normal ZIO Test suite?

Reply:
Removed the non-obviously-linked `RequriesLayer*Spec` that performed the same check.
We'll strictly rely on this test to confirm basic layer sharing now.